### PR TITLE
blueprints/util-test: Add RFC232 variants

### PR DESCRIPTION
**Pull request by snewcomer**
Progress on #15933

### CODE REVIEW

**Review by alexander-alvarez — APPROVED** [review]

File: node-tests/fixtures/util-test/rfc232-unit.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+module('Unit | Utility | foo bar', function(hooks) {
+  setupTest(hooks);
+
+  // Replace this with your real tests.
+  test('it works', function(assert) {
+    let result = fooBar();
+    assert.ok(result);
+  });
+});
```
Comment (Turbo87) [inline node-tests/fixtures/util-test/rfc232-unit.js]: the files should end with a `\n`

**Review by Turbo87 — CHANGES_REQUESTED** [review]

File: blueprints/util-test/qunit-rfc-232-files/tests/unit/utils/__name__-test.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import <%= camelizedModuleName %> from '<%= dasherizedModulePrefix %>/utils/<%= dasherizedModuleName %>';
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+
+module('<%= friendlyTestName %>', function(hooks) {
+  setupTest(hooks);
```
Comment (rwjblue) [inline blueprints/util-test/qunit-rfc-232-files/tests/unit/utils/__name__-test.js]: Hmm. In general I think that we probably don’t need this? I would expect utils to invoke the util method and test its inputs and outputs. Let’s remove `setupTest` here for now?

**Review by rwjblue — CHANGES_REQUESTED** [review]

File: blueprints/util-test/qunit-rfc-232-files/tests/unit/utils/__name__-test.js (hunk 2 above)
Comment (Turbo87) [reply blueprints/util-test/qunit-rfc-232-files/tests/unit/utils/__name__-test.js]: agreed 👍

**Comment by rwjblue** [comment]
Thank you!!

**Pull request merged**